File: kora-app-annotation-processor/src/test/java/ru/tinkoff/kora/kora/app/annotation/processor/app/AppWithInheritanceComponents.java
package ru.tinkoff.kora.kora.app.annotation.processor.app;

import ru.tinkoff.kora.common.KoraApp;

@KoraApp
public interface AppWithInheritanceComponents extends
    AppWithInheritanceComponentsHelper.AppWithInheritanceComponentsModule1,
    AppWithInheritanceComponentsHelper.AppWithInheritanceComponentsModule2,
    AppWithInheritanceComponentsHelper.AppWithInheritanceComponentsModule3 {

}

File: kora-app-symbol-processor/src/test/kotlin/ru/tinkoff/kora/kora/app/ksp/app/AppWithInheritanceComponents.kt
package ru.tinkoff.kora.kora.app.ksp.app

import ru.tinkoff.kora.common.KoraApp

@KoraApp
interface AppWithInheritanceComponents : AppWithInheritanceComponentsHelper.AppWithInheritanceComponentsModule1,
    AppWithInheritanceComponentsHelper.AppWithInheritanceComponentsModule2,
    AppWithInheritanceComponentsHelper.AppWithInheritanceComponentsModule3
